test/models/test_character.js

```diff
@@ -0,0 +1,42 @@
+"use strict";
+
+describe('Character Model', function() {
+    //Clean up after each test.
+    afterEach(function() {
+        simple.restore();
+    });
+
+	describe('Save', function() {
+		it('should save the values.', function() {
+			var ft = new Character();
+			var saved = false;
+			ft.ps.save = function() { saved = true; };
```

Comment:
For this line, in the future, please don't do this. Use mocks instead. Overriding the functions manually isn't fixed by the restore function.

```js
_spy = simple.mock(self.ps, 'save');
_spy.called.should.equal(false);
self.save()
_spy.called.should.equal(true);
```